scripts/core/Config.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Config.h"
+#include "Utils.h"
+#include <fstream>
+#include <algorithm>
+#include <cctype>
+#include <Windows.h> 
+
+bool Config::load(const std::string& filename) {
+    std::ifstream file(filename);
+    if (!file.is_open()) {
+        Logger::Log(L"Could not open config file: " + std::wstring(filename.begin(), filename.end()), Logger::Level::ERROR);
+        return false;
+    }
+
+    std::string line;
+    std::string currentSection;
```

Comment:
use snake_case for the variables like for function or method names, to fit to a convention
```suggestion
    std::string current_section;
```